Source/Contrib/ContentManager/Models/Consist.cs

```diff
@@ -75,18 +75,51 @@ public Consist(Content content)
                         MinCouplerStrengthN = Math.Min(MinCouplerStrengthN, wagonFile.MinCouplerStrengthN);
                         if (wagonFile.MaxBrakeForceN > 0) { NumOperativeBrakes++; }
 
-                        if (wag.IsEngine && engFile.MaxForceN > 25000)  // exclude legacy driving trailers / cab-cars
+                        if (wag.IsEngine)
                         {
-                            EngCount++;
+                            // see MSTSLocomotive.Initialize()
+                            ortsEngAxles = engFile.NumDriveAxles;
+                            if (ortsEngAxles >= 0) { numDriveAxles = ortsEngAxles; }
+                            else if (engFile.NumEngWheels > 7f) { numDriveAxles = (int)(engFile.NumEngWheels / 2f); }
+                            else if (engFile.NumEngWheels > 0) { numDriveAxles = (int)engFile.NumEngWheels; }
```

Comment:
This is inconsistent with the behavior in the simulator. If the number of wheels is >= 7 (not strictly greater than!), then the number of axles is defined to be 4. There's no behavior that divides the number of wheels by 2, so this will produce numbers different from the actual sim.

```suggestion
                            else if (engFile.NumEngWheels != 0 && engFile.NumEngWheels < 7) { numDriveAxles = (int)engFile.NumEngWheels; }
```

That doesn't strictly make sense, but there are years worth of discussions about how useless 'NumWheels' is as a parameter and how huge swaths of content use NumWheels values that are neither correct as the number of wheels nor correct as the number of axles. It's more helpful to tell people what the program will do than to try and make sense of any NumWheels values.

Reply:
Thanks for your feedback. As a new contributor I appreciate the details and insights.

There is one additional difference that you have not commented on and that affects the requested changes. In the parsers, I initialize the ORTS axle counts with -1 (negative one). I did this to know when the ORTS axle counts were absent from the config file. The normal parser defaults to 0 (zero) which cannot be differentiated from a value of 0 (zero).

To accommodate this the checks need to be "> 0" instead of "!= 0".

Are you also concerned with the parser change, and recommend that I revert them?

Reply:
The NumEngWheels count will initialize to 0 according to the setup of EngineFile.cs, don't see anything that needs to be changed about that.